services/QuillLMS/app/controllers/api/v1/active_activity_sessions_controller.rb

```diff
@@ -30,22 +30,10 @@ def update
     head :no_content
   end
 
-  private def working_params
-    return valid_params unless params.dig(:active_activity_session, :passage)
-
-    permitted_passage_object = params[:active_activity_session][:passage].map do |paragraph|
-      paragraph.map do |word_object|
-        word_object.permit!
-      end
-    end
-    valid_params.merge({ passage: permitted_passage_object })
-  end
-
-  private def valid_params
-    params
-      .permit(active_activity_session: {})
-      .require(:active_activity_session)
-      .except(:uid)
+  private def update_params
+    params['active_activity_session']
+      .reject {|key, _| key == 'uid' }
```

Comment:
You can use `except` here.
```suggestion
      .except('uid')
```